stable/chaoskube/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: chaoskube
-version: 3.2.1
-appVersion: 0.14.0
+version: 3.2.2
```

Comment:
Let's bump it to 3.3.0 since the underlying chaoskube version changes quite a bit.